wire-runtime/src/commonMain/kotlin/com/squareup/wire/internal/-Platform.kt

```diff
@@ -24,6 +24,14 @@ expect annotation class Throws(vararg val exceptionClasses: KClass<out Throwable
 
 expect abstract class ObjectStreamException : IOException
 
+expect class ProtocolException(host: String) : IOException
+
 expect fun <T> MutableList<T>.toUnmodifiableList(): List<T>
 
 expect fun <K, V> MutableMap<K, V>.toUnmodifiableMap(): Map<K, V>
+
+internal expect fun String.format(vararg args: Any?): String
+
+internal expect fun Int.intBitsToFloat(): Float
+
+internal expect fun Long.longBitsToDouble(): Double
```

Comment:
Both of these are built-in to Kotlin already as `toBits()`

Reply:
I used `toBits()` for `Float` to `Int` and `Double` to `Long` conversion, but there are no `Int` to `Float` and `Long` to `Double` bits converters. Not sure if `Int.toFloat()` and `Long.toDouble()` are safe to use.

Reply:
There's `fromBits`: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/from-bits.html

Funny enough they're static factories instead of extensions which is funny given my strong argument for extensions today.

Reply:
Since I voted for factories I feel defeated by my own weapon. Rip.